catroid/src/main/java/org/catrobat/catroid/stage/StageLifeCycleController.java

```diff
@@ -242,4 +246,16 @@ static void stageDestroy(StageActivity stageActivity) {
 		}
 		ProjectManager.getInstance().setCurrentlyPlayingScene(ProjectManager.getInstance().getCurrentlyEditedScene());
 	}
+
+	private static void deleteLookFilesToBeDeleted() {
+		for (File file : StageActivity.lookFilesToBeDeleted) {
+			try {
+				StorageOperations.deleteFile(file);
+			} catch (IOException e) {
+				Log.e(TAG, "Error while deleting LookData files marked for deletion"
+						+ " by DeleteLookAction");
+			}
+		}
+		StageActivity.lookFilesToBeDeleted.clear();
+	}
```

Comment:
Please, move this method and the file-set to a companion object in the respective Action.
```suggestion
```